src/SpacetimeDBNetworkManager.cs

```diff
@@ -1,43 +1,54 @@
 #if UNITY_5_3_OR_NEWER
 using System;
 using System.Collections.Generic;
+using System.Linq;
 using SpacetimeDB;
 using UnityEngine;
 
 namespace SpacetimeDB
 {
-	// This class is only used in Unity projects.
```

Comment:
sorry for the whitespace changes here, this file wasn't properly formatted before.

Reply:
reminder that you can view the diff with whitespace turned off
![image](https://github.com/user-attachments/assets/ad3df35a-9ed8-4d42-8fe8-1c7eb3376c57)